arches_her/pkg/preliminary_sql/select_sources.sql

```diff
@@ -1,7 +1,9 @@
 INSERT INTO map_sources(name, source)
     VALUES ('select-application-area', '{
-        "data": "/geojson?nodeid=1909956f-3a3b-11eb-ae99-f875a44e0e11&include_geojson_link=true",
-        "type": "geojson"
+        "data": "/geojson?nodeid=1909956f-3a3b-11eb-ae99-f875a44e0e11", 
+        "type": "vector", 
+        "tiles": ["/application-areas/{z}/{x}/{y}.pbf"], 
+        "minzoom": 6
     }');
 
 INSERT INTO map_sources(name, source)
```

Comment:
`select-area` and `select-monument` are no longer needed and can be removed from the SQL script